Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -711,6 +728,25 @@ bool EventsRefactorer::SearchStringInConditions(
   return false;
 }
 
+bool EventsRefactorer::GetAllSearchableStrings(gd::ObjectsContainer& project,
+                                               gd::ObjectsContainer& layout,
+                                               gd::EventsList& events,
+                                               gd::String search,
+                                               bool matchCase) {
+  // See later for "BuiltinCommonInstructions::Group" when Comment works
+  if (event.GetType() == = "BuiltinCommonInstructions::Comment") {

```

Comment:
No need for this: GetAllSearchableStrings is a method that was added in `BaseEvent`, meaning that *all* the events now have it! 
And it's overriden by CommentEvent, but this method *does not care* who is overriding it.

If in the future, some events is redefining GetAllSearchableStrings, this will work without any changes.

So you can remove this if (also, no triple equal in C++, just ==)